plugins/linux_only.txt

```diff
@@ -1,14 +1,18 @@
 plugins/ipam/dhcp
+plugins/ipam/host-local
```

Comment:
I think these plugins work on windows, so they should not be in this list.

Reply:
But there are plugins like `plugins/ipam/host-local` work both on linux and windows.
What about changing the file name from `linux_only.txt` to `linux.txt`, and `windows_only.txt` to `windows.txt`?
Then add the plugins into the list accordingly, it will be more clarify.

Reply:
if you'd like, feel free. But right now this PR is removing test coverage it shouldn't.

Reply:
Updated with filename changed.
PTAL thanks.